src/components/Common/AvatarEditModal.tsx

```
@@ -318,139 +231,76 @@
-                            </div>
-                          )}
-                        </div>
-                        <p className="text-center font-medium text-secondary-700 mt-2">
-                          {t("preview_cropped_image_hint")}
-                        </p>
-                      </>
-                    )}
+                    <div className="flex h-[50vh] md:h-[75vh] w-full items-center justify-center overflow-scroll rounded-lg border border-secondary-200">
+                      <img
+                        src={
+                          preview && preview.startsWith("blob:")
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)